packages/graphicsmagick.rb

```diff
@@ -30,8 +17,9 @@ class Graphicsmagick < Package
 
   def self.build
     system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
+           "--prefix=/usr/local",
+           "--libdir=/usr/local/lib64",
+           "--mandir=/usr/local/share/man",
```

Comment:
We should be abstracting the install directories.  Does the following work?
```
system "./configure #{CREW_OPTIONS} \
--with-windows-font-dir=#{CREW_PREFIX}/share/fonts/truetype/msttcorefonts \
--with-perl=#{CREW_PREFIX}/bin/perl --disable-maintainer-mode"
```

Reply:
Done